client/my-sites/plans-features-main/test/index.jsx

```diff
@@ -71,7 +71,34 @@ const props = {
 	translate: x => x,
 };
 
+describe( 'PlansFeaturesMain.renderFreePlanBanner()', () => {
+	test.skip( 'Should return null when called with showOnlyEcommercePlans props', () => {
```

Comment:
@jonathansadowski Pointing this out 'cause it's easy to miss--I just added `.skip` for your tests--need to be rewritten for new approach with `planTypes` prop

Reply:
Thanks -- yeah, I noticed that and they're on my list.